### PR TITLE
fix remove-user/group validation

### PR DESCRIPTION
**Pull request by deads2k**
Bug https://bugzilla.redhat.com/show_bug.cgi?id=1225346

Bad validation method.  Found related bug: https://github.com/openshift/origin/issues/2518

@fabianofranz ptal

### CODE REVIEW

**Comment by fabianofranz** [comment]
LGTM

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -503,12 +503,13 @@ osc create -f examples/hello-openshift/hello-pod.json
 osc delete pods hello-openshift
 echo "manage-node: ok"
 
-openshift admin policy add-role-to-group cluster-admin system:unauthenticated
-openshift admin policy remove-role-from-group cluster-admin system:unauthenticated
-openshift admin policy remove-role-from-group-from-project system:unauthenticated
-openshift admin policy add-role-to-user cluster-admin system:no-user
-openshift admin policy remove-role-from-user cluster-admin system:no-user
-openshift admin policy remove-user-from-project system:no-user
+# TODO restore once we have a more reasonable way to create roles
```
Comment (liggitt) [inline hack/test-cmd.sh]: what was wrong with these?

Comment (deads2k) [inline hack/test-cmd.sh]: > what was wrong with these?

Combination of typos, bad cobra command output, and ordering.  Restored.

**Comment by deads2k** [comment]
[merge]

**Pull request merged**